packages/api/src/getNetwork/getNetwork.ts

```diff
@@ -4,9 +4,9 @@ import {
   RequestGetNetworkMessage,
   ResponseType
 } from '@gemwallet/constants';
-import { deserializeError } from '@gemwallet/extension/src/utils/errors';
 
 import { sendMessageToContentScript } from '../helpers/extensionMessaging';
+import { deserializeError } from '../utils/errors';
```

Comment:
Seems that this could actually go to the `helpers` folder

Reply:
Done